app/src/test/java/org/feature/fox/coffee_counter/di/services/network/ApiServiceTest.kt

```diff
@@ -387,7 +389,7 @@ class ApiServiceTest {
 
         mockWebServer.enqueue(response)
 
-        val actualResponse = apiService.purchaseItem(userId, body)
+        val actualResponse = apiService.purchaseItem(bearerToken, userId, body)
```

Comment:
The test cases should also assert if the authorization header was send correctly to mock webserver

Reply:
Should all be resolved in [b11728b](https://github.com/Morphclue/code-camp-2/pull/48/commits/b11728b49f3493cec2fb9a8febf27edd6f1a4243) now.